README.md

```diff
@@ -120,6 +120,319 @@ telegraf --config telegraf.conf --input-filter cpu:mem --output-filter influxdb
 For documentation on the latest development code see the [documentation index](/docs).
```

Comment:
I think you will need a rebase onto master as the README no longer shows all the available plugins.

Reply:
Ok, i'll do it in near future.